script/Ciel/Day 7/07_01_CIEL07_3.txt

```diff
@@ -26,33 +28,34 @@ Today is finally the day I will return to the mansion in time, so Akiha doesn't
 [sha:cc19fe1df3b36c1e60025abbcbfea0373bae5f38]{
 -- Page 352, Offset 15663.
 -- 玄関には翡翠の姿があった。
-I see Hisui standing before the gate.
+// This is the main door of the house, not the gate at the perimeter
+Hisui is standing in front of the main entrance.
 }
 [sha:4b2da9a9e6d54bb2e4ab3801331ac96d4814cf5b]{
 -- Page 352, Offset 15664.
 -- 偶然にしてはタイミングが良すぎる。
-Her timing is too good to be a coincidence.
+Her timing is too perfect for this to be a coincidence.
 }
 [sha:5c2df85d69861c8106767d4281b0d27f76a2d398]{
 -- Page 353, Offset 15665.
 -- 「もしかして、俺が帰ってくるのを待っていてくれたのか？」
-"Wait, were you perhaps waiting for me to return?"
+"Hold on, you weren't waiting here for me to get back were you?"
 }
 [sha:6671047d2588959ddc85838bf6de184d2e12bc1a]{
 -- Page 353, Offset 15666.
 -- 「はい。主人の出迎えは使用人の務めですから」
-"Yes. Greeting their master upon their arrival is a servant's duty."
+"Correct. It is a servant's duty to greet their master upon arrival."
 }
 [sha:ad722cda8b9a87d7eb6fd6d443d10e4c327ba315]{
 -- Page 353, Offset 15667.
 -- ごく当然のように、翡翠は眉ひとつ動かさず返答した。
-Hisui replies without batting an eyelid, as if it was the most normal thing in the world.
+Hisui replies without batting an eyelid, as if this were the most normal thing in the world.
```

Comment:
This reads worse to me, personally

Reply:
I believe this is the correct grammar here - he doesn't believe it's natural, but she's acting as though it is, so this should be subjunctive (were not was)